framework/doc/content/source/postprocessors/AverageVariableChange.md

```diff
@@ -0,0 +1,41 @@
+# AverageVariableChange
+
+This post-processor computes a volume-weighted norm of the change of a variable
+over a time step or a nonlinear iteration:
```

Comment:
let's explcitly call out axuvariables as being an option

I feel like I would do AverageFunctorChange instead. Like you could write such fancy convergence criteria in a ParsedFunctorMaterial then

Reply:
but then again you d have to change base class and posibly duplicate the base class which isnt great

Reply:
Yeah, I'll mention aux variables in the doc. That crossed my mind.

For functors, that crossed my mind too. I just didn't know if there was a functor equivalent of `coupledValuePreviousNL` (or even `coupledValueOld` for that matter) since functors are for on-the-fly calculations.

Reply:
There is not a coupleable api for old fornfunctors

But there is the old state argument. You pass it instead of the current state in the time argument of a functor evaluation